doc/source/methods/PermutationImportance.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
<p>consistency of notation: e_switched -&gt; e_switch ? (check all occurences)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/798/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>